backend/core/src/listings/dto/listing-published-create.dto.ts

```diff
@@ -38,6 +39,12 @@ export class ListingPublishedCreateDto extends OmitType(ListingCreateDto, [
   "reviewOrderType",
   "units",
 ] as const) {
+  @Expose()
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
+  @Type(() => AssetCreateDto)
```

Comment:
I know it's a WIP but i looked at it out of curiosity. Could you put class-validator decorators together and @Type() next to the property definition?